docs/install/tkgs.rst

```diff
@@ -29,9 +29,26 @@ For the deployment on TKG clusters, Kubeflow on vSphere is installed on a Tanzu
 
 - Install ``kctrl``, a kapp-controller's native CLI on your client host. It is used to install  Kubeflow on vSphere Carvel Package. See `Installing kapp-controller CLI: kctrl <https://carvel.dev/kapp-controller/docs/v0.40.0/install/#installing-kapp-controller-cli-kctrl>`__.
 
+Minimus resources for TKG cluster to Install Kubeflow
```

Comment:
(Minimum?)
Suggested: Minimally required resources for TKG cluster to install Kubeflow
(If capital letter, then all capital letter)